core/tests/net/address.cpp

```diff
@@ -0,0 +1,44 @@
+#include <doctest/doctest.h>
+
+#include <cubos/core/net/address.hpp>
+
+TEST_CASE("address")
+{
+    using cubos::core::net::Address;
+
+    SUBCASE("from and to string")
+    {
+        auto addr = Address::from("127.0.0.1");
```

Comment:
Does `Address::from("localhost")` return `127.0.0.1`? If so, we should cover it

Reply:
Same problem as above conversation,